module/VuFind/src/VuFind/Controller/Plugin/ResultScroller.php

```diff
@@ -63,32 +73,49 @@ class ResultScroller extends AbstractPlugin
      */
     protected $resultsManager;
 
+    /**
+     * Search memory
+     *
+     * @var SearchMemory
+     */
+    protected $searchMemory;
+
+    /**
+     * Currently active scroll data
+     *
+     * @var \StdClass
+     */
+    protected $data = null;
+
     /**
      * Constructor. Create a new search result scroller.
      *
      * @param SessionContainer $session Session container
      * @param ResultsManager   $rm      Results manager
      * @param bool             $enabled Is the scroller enabled?
+     * @param SearchMemory     $sm      Search memory
      */
     public function __construct(
         SessionContainer $session,
         ResultsManager $rm,
-        $enabled = true
+        $enabled = true,
+        SearchMemory $sm = null
     ) {
         $this->enabled = $enabled;
 
         // Set up session namespace for the class.
```

Comment:
I'm not sure I understand what this comment means. I wonder if this is a relic from some past design.

Reply:
Likely a relic, yes. I removed it.